requirements.txt

```diff
@@ -0,0 +1,78 @@
+alembic==0.9.6
```

Comment:
Are you sure your project does utilize all these requirements? Ensure only requirements used by this project are here. Virtualenv will help with this.